### ISSUE TITLE
Can't set voucher to earlier date than the default

### ISSUE THREAD

**Issue by makinog3** [filesender/filesender#1193]
When sending a voucher, even if I set a date earlier than the default "Expiry date", the default date will be set.

classes/rest/endpoints/RestEndpointGuest.class.php
```
251         // Set expiry date
252         $expires = $guest->getDefaultExpire();
253         if( $data->expires ) {
254             $expires = max( $expires, $data->expires );
255         }
256         $guest->expires = $expires;
```

**Comment by monkeyiq** [comment]
It might be an idea to have a `min_guest_days_valid` which could be set to -1 to produce the above outcome. Setting `min_guest_days_valid` to a positive value would result in a new user selectable minimum,

```
$expires = $guest->getDefaultExpire();
if( $data->expires ) {
        $expires = max( $data->expires, $guest->getMinimumExpire() );
}
$guest->expires = $expires;
```
...

With getMinimumExpire() returning getDefaultExpire() if `min_guest_days_valid==-1`.

**Comment by monkeyiq** [comment]
In https://github.com/filesender/filesender/pull/1207 I made min_guest_days_valid default to 1 as not having an explicit min was causing edge cases in the code. I don't think anybody would want a guest invitation shorter than a day anyway.

**Comment by WebSpider** [comment]
We have tested this and this is resolved in with the PR

**Comment by yent** [comment]
While this PR has been merged there is no tagged version that includes it. This is a bit of a problem for us, we intended to upgrade but this bug is blocking us from doing so. When can we expect the next 2.x version tag ?

**Comment by monkeyiq** [comment]
As this was merged on Jan 19 it should be in the next release after that time which is https://github.com/filesender/filesender/releases/tag/master-filesender-2.32

It is not ideal that I fixed this issue soon after doing 2.31.

**Comment by monkeyiq** [comment]
Since this has been merged and released I am closing it at this stage. If the issue persists please feel free to reopen.
